assets/css/base/_catro_tabs.scss

```
@@ -0,0 +1,38 @@
+.catro-tabs {
+  .nav-tabs {
+    padding: 0;
+    border-bottom: 1px solid #e0e0e0;
+
+    @media (min-width: 768px) {
```

We should use the mixins. No import needed here.
```suggestion
    @include media-breakpoint-up(md) {
```